hackathon_site/dashboard/frontend/src/components/dashboard/TeamCard/TeamCard.module.scss

```diff
@@ -0,0 +1,36 @@
+// do I have to import the index.scss
+.teamCard {
+    width: 217px;
```

Comment:
Don't hard code the width or height, let those be flexible with whatever container we put them in

Reply:
Imo since we're putting this in a grid container, the width should be dynamic with that.